src/pages/workspaces/migration/WorkspaceItem.test.ts

```diff
@@ -292,26 +338,21 @@ describe('WorkspaceItem', () => {
     // Arrange
     const transferringWorkspace: WorkspaceMigrationInfo = {
       failureReason: undefined,
-      finalBucketTransferProgress: {
-        bytesTransferred: 0,
-        objectsTransferred: 0,
-        totalBytesToTransfer: 0,
-        totalObjectsToTransfer: 0,
-      },
+      finalBucketTransferProgress: undefined,
```

Comment:
We have a different test case above with 0 values, so I wanted to capture the case I had seen in practice where sometimes these are undefined at the beginning of the "transferring" state.